modules/ROOT/pages/notifications/index.adoc

```diff
@@ -1,35 +1,203 @@
+|
+| Successful completion with a regular non-empty result (n > 0 columns, m > 0 rows), for example, a `RETURN` clause with matches.
+
+| 00001
+| Successful completion
+| Omitted result
+| Successful completion with no return columns (n = 0 columns, m = 0 rows), for example, link:https://neo4j.com/docs/cypher-manual/current/planning-and-tuning/[`EXPLAIN` queries].
+
+| 02000
+| No data
+|
+| Successful completion with an empty result (n > 0 columns, m = 0 rows), for example, a `MATCH` clause with no matches.
+|===
+
+GQLSTATUS general codes are filled in by the server unless the server is too old to be aware of GQL-status objects, in which case, it is polyfilled by the driver (see <<server-driver-compatibility, Server - driver compatibility>>).
+The driver tries to deduce `SUCCESS`, `OMITTED RESULT`, or `NO DATA` from the returned number of records and columns.
+If that fails, it uses the GQLSTATUS 02N42 (`no data: unknown subcondition. Unknown GQLSTATUS from old server.`) to indicate that the server did not provide a GQL-status object.
+Then, it poly-fills statuses from the notifications and merges the two lists according to the GQL precedence rules described in <<gqlstatus-notification-object, GQL-status notification object>>.
+
+[[gqlstatus-codes]]
+=== Neo4j-defined GQLSTATUS codes
+
+The Neo4j-defined GQLSTATUS codes are divided into classes and subclasses, where the class code is a 2-character string (one of `00`, `01`, or `03`) and the subclass code is a 3-character string.
+The class code indicates the general condition of the status (such as successful completion, warning, or information), and the subclass code provides more detailed information about the condition, such as classification and messages.
+
+The following table lists the Neo4j-defined groups of GQLSTATUS codes and their meanings:
+
+.GQLSTATUS groups of codes as defined by Neo4j
+[frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
+|===
+|GQLSTATUS code
+|*Description*
+
+| 01N0[y]
+| Deprecation warnings
+
+| 01N3[y]
+| Hint warnings
+
+| 01N4[y]
+| Unsupported warnings
+
+| 01N5[y]
+| Unrecognized warnings
+
+| 01N6[y]
+| Generic warnings
+
+| 01N7[y]
+| Security warnings
+
+| 03N9[y]
+| Performance information
+
+| 03N6[y]
+| Generic information
+
+| 00N5[y]
+| Unrecognized information under successful completion
+
+| 00N6[y]
+| Generic information under successful completion
+
+| 00N7[y]
+| Security information under successful completion
+
+| 00N8[y]
+| Topology information under successful completion
+
+|===
+
+[[notification-grouping-and-filtering]]
+== Server notification grouping and filtering
+
+All server notifications are grouped by category (which is called classification in the GQLSTATUS framework) and severity level, which can be one of `WARNING`, `WARNING OR INFORMATION`, or `INFORMATION`.
+
+The driver-side notification configuration used for filtering notifications by category and severity is the same for both Neo4j and GQLSTATUS notifications.
```

Comment:
As above I don't like the naming 'Neo4j and GQLSTATUS notifications' for the old and new frameworks